### PR TITLE
Add a command to download any SRS files of any size

### PR DESCRIPTION
**Pull request by ppoliani**
Delegate the downloading of SRS files to the end-user. 

Based on the discussion here https://github.com/sigma0-xyz/zkbitcoin/issues/36 and here https://github.com/sigma0-xyz/zkbitcoin/issues/7

### CODE REVIEW

File: src/srs.rs [hunk 1]
```diff
@@ -1,67 +1,49 @@
...
+    Ok(())
 }
 
 /// Returns the local path to the SRS file.
-pub async fn srs_path() -> PathBuf {
-    // download if necessary
-    let srs_path = download_srs().await;
-
-    // always check integrity of file
-    let hash = sha256::try_digest(&srs_path).unwrap();
```
Comment (mimoo) [inline src/srs.rs:62]: IMO we should keep the checks on the digest!

**Review by mimoo — COMMENTED** [review]
Thanks so much for the PR! I also apologize because I know I'm on the hook to review the other ones :)

In addition to the comment I pointed out, I'm wondering if we want to follow this direction. So the question is: do we want to let the application manage the SRS, or do we want the user to do so? In the latter case, we would just ask the user to pass a path to the SRS.

I like the latter path for two reasons:

1. some applications might want to use their own SRS
2. the `request::get` is probably much less optimized than a `wget` for large files (and some of the useful SRS can get really large!)

wdyt?

**Comment by ppoliani** [comment]
@mimoo I don't have a particular preference to be honest. What you say make sense to me. In fact the first version of this PR worked the way you described 😆 

Let me revert some of the commits.

**Comment by ppoliani** [comment]
@mimoo I've removed the new command. Users would have to provide the `srs-path` instead.

File: README.md [hunk 2]
```diff
@@ -26,6 +26,14 @@ To install `snarkjs`, just run:
 npm install -g snarkjs@latest
 ```
 
+### Download SRS File
+
+To create a ZKP you would need to download the correct SRS file (based on your circuit size). For example, if your circuit has around 65K constraints then you would need to download the following file https://storage.googleapis.com/zkevm/ptau/powersOfTau28_hez_final_16.ptau.
```
Comment (mimoo) [inline README.md:31]: sorry for the slow review, we should add a link to https://github.com/iden3/snarkjs#7-prepare-phase-2 which is more user friendly

Comment (mimoo) [reply README.md:31]: also it'd be good to add a `wget` premade command for the lazy asses

**Review by mimoo — APPROVED** [review]
nice! we should probably download that 16 srs in CI for the tests to run properly

**Pull request merged**